tests/inject/dynamic/link-override.tests.ts

```diff
@@ -190,9 +193,9 @@ describe('LINK STYLES', () => {
         expect(getComputedStyle(container.querySelector('h1 strong')).color).toBe('rgb(255, 26, 26)');
     });
 
-    it('should handle styles with invalid url(...)', async () => {
+    xit('should handle styles with invalid url(...)', async () => {
         const importedCSS = 'h1 { background-image: url("freecookies:3https://example.com"); background-color: gray; }';
-        const importedURL = getCSSEchoURL(importedCSS);
+        const importedURL = getCSSDataURL(importedCSS);
```

Comment:
These are the replacement of the echo server.

Reply:
I don't think these should be replaced, using `data:` and having dark reader making a request to a local server are different kind of aspects and by changing this it wouldn't be testing that aspect anymore.

Reply:
Fixed